cli.js

```diff
@@ -78,6 +87,6 @@ process.stdin.on('keypress', (ch, key) => {
 		}
 
 		prevResult = emojis;
-		logUpdate(`${pre}${chalk.bold(query.join(''))}\n${emojis}`);
+		logUpdate(`${pre}${chalk.bold(query.join(''))}\n${emojis}\n`);
```

Comment:
@sindresorhus first of all, thank you very much for guiding me :)

I figured out what you are trying to say. :)